Earthfile

```
@@ -130,3 +130,34 @@ test-integration:
 test:
   BUILD +test-unit
   BUILD +test-integration
+
+# -----------------------Building-----------------------
+
+build:
+  ARG GOARCH=arm64
+  ENV CGO_ENABLED=0 
+  ENV GOARCH=$GOARCH
+  COPY --platform=linux/$USERARCH +go/translate translate
+  WORKDIR translate
+  RUN \
+  --mount=type=cache,target=/go/pkg/mod \
+  --mount=type=cache,target=/root/.cache/go-build \
+    go build -o translate cmd/translate/main.go
+  SAVE ARTIFACT translate bin/translate
+
+image:
+  ARG TARGETARCH
+  ARG --required registry
+  ARG tag=latest
+  FROM alpine
+  COPY --platform=linux/$USERARCH (+build/bin/translate --GOARCH=$TARGETARCH) /translate
+  ENTRYPOINT ["/translate"]
+  SAVE IMAGE --push $registry/translate:$tag
+
+image-multiplatform:
+  ARG --required registry
+  BUILD \
+  --platform=linux/amd64 \
+  --platform=linux/arm/v7 \
+  --platform=linux/arm64 \
```

Let's support the following initially!

```suggestion
  --platform=linux/amd64 \
  --platform=linux/arm64 \
```